newrelic/resource_newrelic_alert_channel.go

```diff
@@ -262,3 +288,15 @@ func resourceNewRelicAlertChannelDelete(d *schema.ResourceData, meta interface{}
 
 	return nil
 }
+
+// StripWhitespace removes whitespace from a string.
+func StripWhitespace(str string) string {
```

Comment:
Thoughts on moving this to `helpers.go`? Could add a unit test for it.

Reply:
👍